docs/Installation Guide.md

```diff
@@ -0,0 +1,319 @@
+6. [Using Vagrant](https://github.com/Nerdylicious/app-web-server/blob/master/docs/Installation%20Guide.md#using-vagrant)
+7. [Install Project Dependencies](https://github.com/Nerdylicious/app-web-server/blob/master/docs/Installation%20Guide.md#install-project-dependencies)
+8. [Setup PostgreSQL](https://github.com/Nerdylicious/app-web-server/blob/master/docs/Installation%20Guide.md#setup-postgresql)
+9. [Update settings.py](https://github.com/Nerdylicious/app-web-server/blob/master/docs/Installation%20Guide.md#update-settingspy)
+10. [Generate Database Tables Corresponding to Django Models](https://github.com/Nerdylicious/app-web-server/blob/master/docs/Installation%20Guide.md#generate-database-tables-corresponding-to-django-models)
+11. [Run Development Server](https://github.com/Nerdylicious/app-web-server/blob/master/docs/Installation%20Guide.md#run-development-server)
+12. [Try out Mobile App Control Center](https://github.com/Nerdylicious/app-web-server/blob/master/docs/Installation%20Guide.md#try-out-mobile-app-control-center)
+13. [Exit the Virtual Machine](https://github.com/Nerdylicious/app-web-server/blob/master/docs/Installation%20Guide.md#exit-the-virtual-machine)
+
+## Install git
+
+If you don't already have git, you can download it [here](http://git-scm.com/downloads). You must then install git.
+
+## Clone Project
+
+Clone the project from GitHub by running the following command:
+
+    git clone project_url_here
+
+For my project, this would correspond to:
+
+    git clone git@github.com:Nerdylicious/app-web-server.git
+
+## Install Django and PostgreSQL
+
+If you are installing and running the project on your local machine and not on the PeaceCorps VM, then you will need to download and install the following software:
+
+1. [Django](https://www.djangoproject.com/download/) (version >= 1.6.5)
+2. [PostgreSQL](http://www.postgresql.org/download/) (version >= 9.3.4)
+
+**Skip this step if you are installing and running the project on the PeaceCorps VM, as Django and PostgreSQL are already included in the PeaceCorps VM.**
+
+## Install VirtualBox and Vagrant
+
+The Virtual Machine provided by PeaceCorps will be downloaded in a later step. We must first install VirtualBox and Vagrant.
+
+The VirtualBox and Vagrant installers can be downloaded from the following websites:
+
+1. [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+2. [Vagrant](http://www.vagrantup.com/downloads.html)
+
+Install VirtualBox and Vagrant by running the installers.
+
+## Download PeaceCorps Ubuntu Virtual Machine
+
+A Vagrant file is located in the top level directory for the project (at [https://github.com/Nerdylicious/app-web-server/blob/master/Vagrantfile](https://github.com/Nerdylicious/app-web-server/blob/master/Vagrantfile) found on GitHub. In case you do not have a copy of this Vagrant file, here are it's contents:
```

Comment:
Is the content of vagrantfile needed here? if somebody is checking out the repo .. they will have it right? May be we  could just point to the location of the vagrantfile  and ask them to do "vagrnat up"? I do not feel strongly about it .. just a suggestion